src/pages/Admin/Room/Edit/RoomEditModal.tsx

```diff
@@ -183,6 +183,24 @@ export const RoomEditModal: React.FC<RoomEditModalProps> = ({
             </div>
           </Form.Row>
 
+          <Form.Row>
+            <div className="room-edit-modal__input">
+              <Form.Label>Change room type</Form.Label>
+              <Form.Control
+                as="select"
+                custom
+                name="type"
+                ref={register}
+                defaultValue={room.type}
+              >
+                <option />
+                {Object.values(RoomTypes).map((roomType) => (
+                  <option key={roomType}>{roomType}</option>
+                ))}
```

Comment:
Do we also need to set the `selected` prop on one of these? Or does `Form.Control` handle that for us?

**Further Reading:**

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option#attributes

Reply:
The component handles the setting of the selected prop.

Reply:
Nice